librz/analysis/class.c

```diff
@@ -878,10 +878,23 @@ RZ_API RzVector /*<RzAnalysisVTable>*/ *rz_analysis_class_vtable_get_all(RzAnaly
 }
 
 RZ_API RzAnalysisClassErr rz_analysis_class_vtable_set(RzAnalysis *analysis, const char *class_name, RzAnalysisVTable *vtable) {
+	/* Check if vtable exists before setting it */
+	RzVector /*<RzAnalysisVTable>*/ *vtables = rz_analysis_class_vtable_get_all(analysis, class_name);
```

Comment:
you should free this vector because it is dynamically allocated.

Reply:
sorry for my cheap mistake..

Reply:
No problem :)